frontend/src/App.tsx

```diff
@@ -70,6 +71,13 @@ function App() {
     initializeOpenAPI(loginState, i18n);
   }, [loginState, i18n]);
 
+  // Plausible will not make HTTP requests to track events when
+  // `localtion.hostname` is "localhost".
+  const plausible = Plausible({
+    apiHost: process.env.REACT_APP_WEBSITE_ANALYTICS_URL,
+  });
+  plausible.enableAutoPageviews();
```

Comment:
What if `process.env.REACT_APP_WEBSITE_ANALYTICS_URL` is empty or not defined ?  
I would find clearer to wrap this initialization into a condition to check that the configuration is present.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1340/commits/e2d4dc05ea9be7d63636029aff7f3be0f498311e.

`enableAutoPageviews` and `trackEvent` are now wrapped inside a `if (client)` condition. The variable `client` is defined only when `REACT_APP_WEBSITE_ANALYTICS_URL` is defined and not empty.